tools/dependency_summary.py

```diff
@@ -742,6 +736,17 @@
             "fixup": "https://raw.githubusercontent.com/LeopoldArkham/humansize/master/LICENSE-APACHE",
         }
     },
+
+    "encoding_rs": {
```

Comment:
Explanation of this fixup in https://github.com/hsivonen/encoding_rs/commit/3a4033e67b6b9d1c1e9514bcb5c20ae05bf8391d

It's an interesting case, and I'm not an expert in licensing so I would love to have someone take a look here to make sure what I'm doing is fine

what I end up doing is adding this as a special case in our licenses, and it gets listed on it own, for example: https://github.com/mozilla/application-services/pull/4729/files#diff-c0246cf339c103dac88ae8c472de424275028137e404e05ee6c54e7f9bc1f45bR1926-R1951

This seems OK to me, since we are listing the license shown on the project as-is, but  would love a second opinion on this